docs/run/primitives-get-started.mdx

```diff
@@ -38,8 +38,6 @@ backend = service.least_busy(operational=True, simulator=False, min_num_qubits=1
 
 You need at least one circuit and one observable as inputs to the `Estimator` primitive.
 
-<Tabs>
-  <TabItem value="EstimatorV2" label="V2 Estimator">
 ```python
 import numpy as np
 from qiskit.circuit.library import IQP
```

Comment:
Given that the goal of this PR is to improve readability, would it make sense to choose a more intuitive circuit for the code example?? Having to import IQP and `random_hermitian` looks a bit cumbersome, I'd go for a manually built circuit.

Reply:
Hi Elena, this PR is just for improving the formatting, not the content, which should be updated in a separate PR.